cmd/metal-api/internal/datastore/switch.go

```diff
@@ -196,3 +196,39 @@ func (rs *RethinkStore) ConnectMachineWithSwitches(m *metal.Machine) error {
 
 	return nil
 }
+
+// GetSwitchStatus get SwitchStatus for a given switch id
+func (rs *RethinkStore) GetSwitchStatus(id string) (*metal.SwitchStatus, error) {
+	rs.log.Infow("get state", "id", id)
```

Comment:
These kinds of logs should be debug only. metal-api primarily logs requests only.

Reply:
done